templates/containers/chiseled/index.html

```diff
@@ -511,7 +511,7 @@ <h3 class="p-text--small-caps">Documentation</h3>
                 <a href="https://documentation.ubuntu.com/rockcraft/en/latest/how-to/rocks/chisel-existing-rock/">Using chisel in Rockcraft</a>
               </li>
               <li class="p-list__item">
-                <a href="https://documentation.ubuntu.com/oci-registries/en/latest/oci-how-to/create-chiseled-ubuntu-image/">Create
+                <a href="https://canonical-public-cloud.readthedocs-hosted.com/en/latest/oci/oci-how-to/create-chiselled-ubuntu-image.html">Create
```

Comment:
this link goes through several redirects and ends up at https://documentation.ubuntu.com/chisel/en/latest/tutorial/getting-started/. Why not use the end link instead?

Reply:
ah, in that case that's fine, let's leave it like that